web/templates/goal/show.html.eex

```diff
@@ -3,7 +3,7 @@
   <div class="ma3 ma4-m ma4-l mv2 mv3-m mv4-l">
     <b class="di"><%= markdown String.trim_trailing(@goal.content, " #Goal") %></b>
     <%= link "Edit", to: goal_path(@conn, :edit, @goal), class: "di" %>
-    <p class="mv3">Created <%= @goal.inserted_at %></p>
+    <p class="mv3">Created <%= String.slice(Date.to_string(@goal.inserted_at), 0..10) %></p>
```

Comment:
Ahh ok, I'd still take a look at the [Timex](https://github.com/bitwalker/timex) library which I'm already using as it allows for flexible date formatting.